docs/package.json

```diff
@@ -6,6 +6,7 @@
     "dev": "docusaurus start --port 3200 --no-open",
     "docusaurus": "docusaurus",
     "start": "docusaurus start --port 3200",
+    "gen": "ts-node -r tsconfig-paths/register ./scripts/gen-event-webhook-doc.ts",
```

Comment:
* Can this be generated each time a dependent file changes?
* Add result in the git repo/diff.

Reply:
Add pre-commit hook to regenerate.